test/PushNotificationTests/APITests.cpp

```diff
@@ -36,36 +38,56 @@ namespace Test::PushNotifications
             TEST_CLASS_PROPERTY(L"RunAs:Class", L"RestrictedUser")
         END_TEST_CLASS()
 
+        static std::wstring GetDeploymentDir()
+        {
+            WEX::Common::String testDeploymentDir;
+            WEX::TestExecution::RuntimeParameters::TryGetValue(L"TestDeploymentDir", testDeploymentDir);
+            return reinterpret_cast<PCWSTR>(testDeploymentDir.GetBuffer());
+        }
+
+        static std::filesystem::path GetTestVCLibsPackageFile()
+        {
+            auto filename{ std::filesystem::path(GetDeploymentDir()) };
+            filename /= L"VCLibs.appx";
+            return filename;
+        }
+
+        static std::filesystem::path GetTestPackageFile()
+        {
+            auto filename{ std::filesystem::path(GetDeploymentDir()) };
+            filename /= L"PushNotificationsTestAppPackage.msix";
+            return filename;
+        }
+
+        static std::wstring GetTestPackageFullName()
+        {
+            return L"PushNotificationsTestAppPackage_1.0.0.0_" PROJECTREUNION_TEST_PACKAGE_DDLM_ARCHITECTURE L"__8wekyb3d8bbwe";
+        }
+
         TEST_CLASS_SETUP(ClassInit)
         {
             try
             {
-                InstallPackage(c_testVCLibsPackageFile);
+                TP::AddPackage_ProjectReunionFramework();
+                InstallPackage(GetTestVCLibsPackageFile());
+                InstallPackage(GetTestPackageFile());
             }
             catch (...)
             {
                 return false;
             }
 
-            TP::AddPackage_ProjectReunionFramework();
-            try
-            {
-                InstallPackage(c_testPackageFile);
-            }
-            catch (...)
-            {
-                return false;
-            }
+            m_testAppLauncher = winrt::create_instance<IApplicationActivationManager>(CLSID_ApplicationActivationManager, CLSCTX_ALL);
+            VERIFY_IS_NOT_NULL(m_testAppLauncher);
```

Comment:
Do we need this check? I would expect that either we get an instance or we throw in the previous line.